### ISSUE TITLE
PMID:22918954 mid1

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3661]
for

fig 3 B
with misplaced (arrow), misoriented (arrowhead), and curved (star) septa. 

misplaced is "asymmetric" (i.e much closer to one end of cell)
we do have

mislocalized septum, near new end
but I don't know  if it is old or new end.

misoriented, i used   "mislocalized, misoriented septum"

**Comment by mah11** [comment]
What is the actual request here?

Is there any reason not to use these existing terms?

mislocalized septum during vegetative growth (FYPO:0000339)
misoriented septum during vegetative growth (FYPO:0001390)

**Comment by ValWood** [comment]
Sorry, crap ticket, bu I got a bit confused here.

mislocalized septum during vegetative growth (FYPO:0000339)
has 2 children
mislocalized septum, near new end  
mislocalized, misoriented septum  

so for the misoriented septum I used
mislocalized, misoriented septum  

then to differntiate the  septa that were misplaced but not misoriented 
I wanted to use a sibling.

But "mislocalized septum, near new end"
was not appropriate.

maybe I just leave it at
mislocalized septum during vegetative growth (FYPO:0000339)
it will be fine, because it's that same allele, it's not as if there is anything radically different going on  in these different placements.

so it's ok
